joomla/joomla-cms
# [com_menus] Add missing tooltip in search

#### Summary of Changes

Simple PR to add a missing tootip (menus view) in the com_menus search filters.
Also improves a little the tooltip on items view and element label (accessibility).
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Menus -> Manage, check the tooltip in the search field.
3. Go to Menus -> [Select one menu], check the tooltip in the search field.
